source/slang/slang-ir-use-uninitialized-values.cpp

```diff
@@ -0,0 +1,315 @@
+#include "slang-ir-use-uninitialized-values.h"
+#include "slang-ir-insts.h"
+#include "slang-ir-reachability.h"
+#include "slang-ir.h"
+
+namespace Slang
+{
+    bool metaOp(IRInst* inst) {
```

Comment:
Coding style for this project is new line for braces with few exceptions.
```suggestion
    bool metaOp(IRInst* inst) 
    {
```